src/clarity-angular/wizard/wizard-button.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {
+    Component,
+    Input,
+    Output,
+    EventEmitter
+} from "@angular/core";
+
+import { WizardNavigationService } from "./providers/wizard-navigation";
+import { ButtonHubService } from "./providers/button-hub";
+
+export const DEFAULT_BUTTON_TYPES: any = {
+    cancel: "cancel",
+    previous: "previous",
+    next: "next",
+    finish: "finish",
+    danger: "danger"
+};
+
+export const CUSTOM_BUTTON_TYPES: any = {
+    cancel: "custom-cancel",
+    previous: "custom-previous",
+    next: "custom-next",
+    finish: "custom-finish",
+    danger: "custom-danger"
+};
```

Comment:
you can designate a button as "custom-finish" and it will inherit the look and feel (as well as the functionality making it available, unavailable, enabled, and disabled) as a regular finish button.

it's a convenience for people needing custom buttons.